### PR TITLE
Added release notes

### PR DESCRIPTION
**Pull request by glo82145**
## Description

Release notes for PWA Studio 13.2.0

## Related Issue

Closes #https://jira.corp.adobe.com/browse/PWA-3146

## Acceptance

- @supernova-at 
- @glo42707 

### Verification Stakeholders

- @tkacheva

## Verification Steps

Check changlog.md for accuracy.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,32 +1,45 @@
...
+**NOTE:**
+_This changelog only contains release notes for PWA Studio and Venia 13.2.0_
 _For older release notes, see_ [PWA Studio releases][].
 
 ## Highlights
 
-- **PHP 8.2 support** — PWA Studio now supports PHP 8.2. GitHub PRs: [4][], [25][], [34][], [5][], [20][], [10][]
-- **Payment method persists** — The chosen payment method now persists throughout the user's session. GitHub PR: [3969][]
-- **Node 18 support** — PWA Studio now supports Node 18.0,0+. GitHub PR: [172][]
+The 13.0 release of PWA Studio provides customers with many accessibility improvements for keyboard and screen-reader access, as well as several bug fixes. Full details are described below.
```
Comment (supernova-at) [inline CHANGELOG.md:9]:
```suggestion
The 13.2 release of PWA Studio ...
```

Comment (supernova-at) [inline CHANGELOG.md:9]: Should say the `13.2.0` release of PWA Studio

**Review by supernova-at — CHANGES_REQUESTED** [review]
One minor fix but other than that it looks great!

**Review by glo42707 — DISMISSED** [review]
approved

**Review by supernova-at — DISMISSED** [review]

**Comment by glo82145** [comment]
run lighthouse-desktop

**Comment by glo82145** [comment]
run lighthouse-mobile

**Comment by glo42707** [comment]
run lighthouse-desktop

**Review by glo42707 — APPROVED** [review]
approving for develop PR Merge test

**Review by glo42707 — APPROVED** [review]
approving again for release/13.2 branch test if able to merge

**Pull request merged**